test/GitcoinGovernor.t.sol

```diff
@@ -1086,3 +1084,226 @@ contract NewGitcoinGovernorProposalTest is GitcoinGovernorProposalTestHelper {
+    );
+
+    ( // Ensure the new votes were recorded.
+    _againstVotesCast, _forVotesCast, _abstainVotesCast) =
+      governorBravo.proposalVotes(newProposalId);
+    assertEq(_firstVote.forVotes + _secondVote.forVotes, _forVotesCast);
+    assertEq(_firstVote.againstVotes + _secondVote.againstVotes, _againstVotesCast);
+    assertEq(_firstVote.abstainVotes + _secondVote.abstainVotes, _abstainVotesCast);
+
+    // Confirm nominal votes can co-exist with partial+fractional votes by
+    // voting with the second largest delegate.
+    uint256 _nominalVoterWeight =
+      gtcToken.getPriorVotes(delegates[1], governorBravo.proposalSnapshot(newProposalId));
+    vm.prank(delegates[1]);
+    governorBravo.castVote(newProposalId, FOR);
+
+    ( // Ensure the nominal votes were recorded.
+    _againstVotesCast, _forVotesCast, _abstainVotesCast) =
+      governorBravo.proposalVotes(newProposalId);
+    assertEq(_firstVote.forVotes + _secondVote.forVotes + _nominalVoterWeight, _forVotesCast);
+    assertEq(_firstVote.againstVotes + _secondVote.againstVotes, _againstVotesCast);
+    assertEq(_firstVote.abstainVotes + _secondVote.abstainVotes, _abstainVotesCast);
+  }
+
+  function testFuzz_ProposalsCanBePassedWithSplitVotes(
+    uint256 _forVotePercentage,
+    uint256 _againstVotePercentage,
+    uint256 _abstainVotePercentage
+  ) public {
+    _forVotePercentage = bound(_forVotePercentage, 0.0e18, 1.0e18);
+    _againstVotePercentage = bound(_againstVotePercentage, 0.0e18, 1.0e18 - _forVotePercentage);
+    _abstainVotePercentage =
+      bound(_abstainVotePercentage, 0.0e18, 1.0e18 - _forVotePercentage - _againstVotePercentage);
+
+    uint256 _weight = gtcToken.getPriorVotes(
+      PROPOSER, // The proposer is also the top delegate.
+      governorBravo.proposalSnapshot(newProposalId)
+    );
+
+    uint128 _forVotes = uint128(_weight.mulWadDown(_forVotePercentage));
+    uint128 _againstVotes = uint128(_weight.mulWadDown(_againstVotePercentage));
+    uint128 _abstainVotes = uint128(_weight.mulWadDown(_abstainVotePercentage));
+
+    // The accepted support types for Bravo fall within [0,2].
+    uint8 _supportTypeDoesntMatterForFlexVoting = uint8(2);
+
+    vm.prank(PROPOSER);
+    governorBravo.castVoteWithReasonAndParams(
+      newProposalId,
+      _supportTypeDoesntMatterForFlexVoting,
+      "My vote",
+      abi.encodePacked(_forVotes, _againstVotes, _abstainVotes)
+    );
+
+    ( // Ensure the votes were split.
+    uint256 _actualAgainstVotes, uint256 _actualForVotes, uint256 _actualAbstainVotes) =
+      governorBravo.proposalVotes(newProposalId);
+    assertEq(_forVotes, _actualForVotes);
+    assertEq(_againstVotes, _actualAgainstVotes);
+    assertEq(_abstainVotes, _actualAbstainVotes);
+
+    _jumpToVotingComplete(newProposalId);
+
+    IGovernor.ProposalState _state = governorBravo.state(newProposalId);
+
+    if (_forVotes >= governorBravo.quorum(block.number) && _forVotes > _againstVotes) {
```

Comment:
Is there a method that checks if quorum is reached? Seems we're verifying that manually but we might change quorum so want to just make sure we don't forget about this at that time. Just mentioning this part in #18 is sufficient. Feel free to resolve this comment yourself.

Reply:
Good call out. Added a note to #18 about this. Thanks!